pkg/synchronizer/post.go

```diff
@@ -38,16 +38,18 @@ func (p *RESTPusher) PushUpdate(endpoint string, data []byte) error {
 
 	log.Infof("Push Update endpoint=%s data=%s", endpoint, string(data))
 
-	resp, err := client.Post(
-		endpoint,
-		"application/json",
-		bytes.NewBuffer(data))
 
-	/* In the future, PUT will be the correct operation
-	resp, err := httpPut(client, endpoint, "application/json", data)
-	*/
+	req, err := http.NewRequest(http.MethodPut, endpoint, bytes.NewBuffer(data))
```

Comment:
This looks like it is dependent on some version of SD-CORE - does this break backward compatibility?